.github/workflows/tests-nightly.yml

```diff
@@ -155,6 +155,14 @@ jobs:
       TOKEN: ${{ secrets.CLOUD_ACCESS_TOKEN }}
       VERSION: ${{github.event.client_payload.version}}
     steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

Comment:
A bit weird that we have `actions/checkout@v1` at line 103 while the rest of the workflow is using `v3`